sales_tax_solution.rb

```diff
@@ -0,0 +1,66 @@
+# general
+products = ["perfume", "Music CD"]
+medical_products =["headache", "pills"]
+food_products = ["Chocolate", "Chocolates", "chocolate", "chocolates"]
```

Comment:
What is the reason behind making food_products and medical_products when ultimately we are using exempted_products